configs/samples/pjsip.conf.sample

```diff
@@ -1015,7 +1015,18 @@
                 ; using the same CHANNEL function if needed. Setting tenant ID here
                 ; will cause it to show up on channel creation and the initial
                 ; channel snapshot.
-
+;
+; suppress_moh_on_remote_hold = no
+                ; Normally, when Alice calls Bob and Bob sends Asterisk a
```

Comment:
This is not accurate, as this extends beyond just a 183. It is during any SDP negotiation including before the call is answered or when it is answered - whether the "m" option is used or not.

Reply:
Of course.  I only documented the case that brought this up to keep the description simple.  I'll update it though.